src/server/devHolocronCDN.js

```diff
@@ -18,36 +18,20 @@
 
 import path from 'path';
 import fs from 'fs';
-import express from 'express';
-import cors from 'cors';
 import { argv } from 'yargs';
-import Fastify from 'fastify';
-import fastifyExpress from '@fastify/express';
-import oneAppDevCdn from '@americanexpress/one-app-dev-cdn';
+import oneAppDevCdn from './utils/devCdnFactory';
```

Comment:
do we really need the business logic in `utils/devCdnFactory`? why can't we just put all the code from `utils/devCdnFactory` in this file?